guides/07_streaming/04_conversational-chatbot.md

```diff
@@ -0,0 +1,187 @@
+# Building Conversational Chatbots with Gradio
+
+Tags: AUDIO, STREAMING, CHATBOTS
+
+## Introduction
+
+The next generation of AI user interfaces is moving towards audio-native experiences. Users will be able to speak to chatbots and receive spoken responses in return. Several models have been built under this paradigm, including GPT-4o and [mini omni](https://github.com/gpt-omni/mini-omni).
+
+In this guide, we'll walk you through building your own conversational chat application using mini omni as an example. You can see a demo of the finished app below:
+
+[omni mini demo](https://github.com/user-attachments/assets/db36f4db-7535-49f1-a2dd-bd36c487ebdf)
```

Comment:
Can we embed the video instead of linking it?

Reply:
Yea this is a typo. Thank you!

Reply:
did this get fixed?

Reply:
Now it did 😄